vue-components/src/components/LookupMenu.vue

```diff
@@ -107,12 +118,14 @@ $base: '.wikit-LookupMenu';
 		transition-duration: $wikit-LookupMenu-item-transition-duration;
 		transition-timing-function: $wikit-LookupMenu-item-transition-timing-function;
 
-		&:hover {
+		&:hover,
+		&--selected {
 			background-color: $wikit-LookupMenu-item-hover-background-color;
 			cursor: $wikit-LookupMenu-item-hover-cursor;
 		}
 
-		&:active {
+		&:active,
+		&--active {
```

Comment:
So I think we had a misunderstanding here, I think the blue color we had before should still be included but only on mouse click (`:active`, currently it isn't working), also if all this `hasActiveElement` property does is try to mimic an active mouse mode, then I'm pretty sure we do not need it. I don't think I ever saw an implementation of `:active` styles mimicked through the keyboard enter. But maybe that's what @SaiSan-WMDE intended?

Reply:
`:active` is working fine for me (checking in Chrome). I've seen implementations of `:active`styles that are persistent on keyboard enter and thought that was the default way to go, but unfortunately I failed to notice that OOUI is not one of them, so I guess we don't need the `hasActiveElement` property in case it's only trying to emulate the active state. Sorry about that

Reply:
Ah, sorry. Then I was misled by the statements that the component should behave (and to me that implied look) the same whether selecting an item on click or on Enter.

Reply:
Oddly enough, on Firefox it works on Enter down but not on Mouse down :man_shrugging: Works as expected on Firefox. And don't get me wrong, if the intention was to have the mouse down behaviour on enter, then by all means, keep it. I don't think we need to have a 1:1 facsimile of OOUI, I really was just unclear on the design intent.

Reply:
Based on @SaiSan-WMDE comment. I just removed the whole  hasActiveElement thing. Now the active style is only applied up mouse click. there will be no 'active' style when a user selects using Enter.
@itamargiv There is a separate ticket to address the browser issues that you're currently experiencing. 
see here: https://phabricator.wikimedia.org/T265917

that will be my next task as soon as we're done with this one.

Reply:
Thank you @bereket-WMDE Totally missed it.